src/main/java/duke/operations/Ui.java

```diff
@@ -0,0 +1,41 @@
+package duke.operations;
+
+/**
+ * Deals with interactions with the user.
+ */
+public class Ui {
+    private static final String WELCOME_MESSAGE_ONE = "Tell me... have you seen a RED imposter among us?";
+    private static final String WELCOME_MESSAGE_TWO = "If you have seen this SUSSY imposter,"
+            + " please let me know immediately... otherwise how may I be of assistance?";
+    private static final String GOODBYE_MESSAGE = "Better watch out for the imposter AMONG US!";
+
+    /**
+     * Prints a welcome message.
+     *
+     * @return the welcome message for Duke.
+     */
+    public static String showWelcome() {
+        String getWelcomeMsgOne = WELCOME_MESSAGE_ONE;
+        String getWelcomeMsgTwo = WELCOME_MESSAGE_TWO;
+        return getWelcomeMsgOne + "\n" + getWelcomeMsgTwo;
```

Comment:
Can't you just return WELCOME_MESSAGE_ONE + "\n" + WELCOME_MESSAGE_TWO? Why create two local vars?